### ISSUE TITLE
Connection lost after 10 min or so on wifi and connects to wifi only after reboot

### ISSUE THREAD

**Issue by rkp600** [lwfinger/rtlwifi_new#70]
Here is my syslog
http://pastebin.com/YQ7TzZid.

**Comment by lwfinger** [comment]
Your log is showing that the interface is using the "direct probe" method of connection. That usually happens only when the signal is very week. Is that your case?

What options are you using when you load the module?

**Comment by rkp600** [comment]
Some times it's weak but when I see a windows system, it's remain connected to network for long time whereas mine disconnects.For loading module I use network manager applet.

**Comment by saksham0808** [comment]
This thing happens with me off and on. I install these drivers to reduce this thing and it works good quite often. But it does come back to annoy after a while.

**Comment by alfbr** [comment]
I have been experiencing this issue too. The signal is decent, but not excellent. Here is output from a scan:
          Cell 01 - Address: D6:CA:6D:11:91:38
                    Channel:4
                    Frequency:2.427 GHz (Channel 4)
                    Quality=28/70  Signal level=-82 dBm  
                    Encryption key:on
                    ESSID:"gjest"
                    Bit Rates:1 Mb/s; 2 Mb/s; 5.5 Mb/s; 11 Mb/s; 6 Mb/s
                              9 Mb/s; 12 Mb/s; 18 Mb/s
                    Bit Rates:24 Mb/s; 36 Mb/s; 48 Mb/s; 54 Mb/s
                    Mode:Master
                    Extra:tsf=00000111dbffaece
                    Extra: Last beacon: 0ms ago
                    IE: Unknown: 0005676A657374
                    IE: Unknown: 010882848B960C121824
                    IE: Unknown: 030104
                    IE: Unknown: 07064E5A20010B1B
                    IE: Unknown: 2A0104
                    IE: Unknown: 32043048606C
                    IE: IEEE 802.11i/WPA2 Version 1
                        Group Cipher : CCMP
                        Pairwise Ciphers (1) : CCMP
                        Authentication Suites (1) : PSK
                    IE: Unknown: 2D1ACC111BFFFF000000000000000000000100000000000000000000
                    IE: Unknown: 3D1604000400000000000000000000000000000000000000
                    IE: Unknown: DD180050F2020101000003A4000027A4000042435E0062322F00

Any thing I can do to bug hunt?

**Comment by alfbr** [comment]
BTW, I see the signal strength is fluctuating quite a bit, just did a scan again and signal is 64/70 with level -46 dBm.

**Comment by alfbr** [comment]
Sorry forgot to mention loading options. After the issue surfaced I added the ips=0 option, so I now have both msi= and ips=0.

**Comment by lwfinger** [comment]
Differences in the signal strength are not usually that large. Is there something unique to your physical setup? Please describe it.

**Comment by alfbr** [comment]
I am actually having trouble reproducing the low signal strength from yesterday. Right now I am getting 70/70 with -40 dBm (same location, same set-up). I am sitting in a room about ten meters away from the router. The router has been running well for a couple of years, and is used by several devices successfully. This router (is actually not the same router I used in the previous issue this week-end, but also) runs openwrt with a well supported atheros wireless chip.

After reporting the issue, I re-compiled the rtl8723be driver (in case a kernel update somehow caused problems). While the disconnect (with a reboot necessary for re-connecting) was frequent yesterday, I have not seen it since re-compiling the driver. I have now turned off software updates to keep the system pristine. Will report back if or when I am able to reproduce.

**Comment by lwfinger** [comment]
A completely new version of the drivers has just been pushed. It fixes a lot of things. As a result, I am closing all open issues. If you get a new one, please create a new issue.

Note: The new driver fixes the problems with power save. It is recommended that you clear all ips, swlps, and fwlps options.
